Dockerfile

```diff
@@ -67,6 +70,12 @@ RUN wget "https://github.com/digitalocean/doctl/releases/download/v${DOCTL_VERSI
   && chmod +x /usr/local/bin/doctl \
   && doctl version | grep -q "${DOCTL_VERSION}"
 
+ARG RUBY_VERSION=3.0
+## Always use the latest Gem versions
+# hadolint ignore=DL3028
+RUN apk add --no-cache ruby=~"${RUBY_VERSION}" \
```

Comment:
Does this mean we'll have to update this Dockerfile every time we're using a new gem in infra-reports or any other repo?

Reply:
The infra-reports Dockerfile contains instructions to add gems, not sure it will ve enought, if yes then we should remove all installed gems here IMO.